fachhochschule-sudwestfalen.csl

```diff
@@ -88,14 +86,20 @@
   </macro>
   <macro name="year-date">
     <choose>
-      <if variable="issued">
-        <date variable="issued">
-          <date-part name="year"/>
-        </date>
+      <if match="none" position="ibid">
```

Comment:
This doesn't look right: the year-date macro is used in the Bibliography, where the test doesn't apply

Reply:
my bad - I think I fixed it now.